configs/server_hello.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "manifest": "s3://goployer/manifest/hello.yaml",
+    "manifest_s3_region": "ap-northeast-2",
+    "ami": "",
+    "env": "",
+    "stack": "artd",
+    "assume_role": "",
+    "region": "ap-northeast-2",
+    "slack_off": true,
+    "log_level": "debug",
+    "extra_tags": "",
+    "ansible_extra_vars": "",
+    "override_instance_type": "t3.small",
+    "disable_metrics": false,
+    "release_notes": "test",
+    "release_notes_base64": "",
+    "force_manifest_capacity": false,
+    "polling_interval": 60
```

Comment:
Maybe it should be `60s`. "polling_interval: 60" means `60 Nanoseconds`

Reply:
Yeah!! I am worried about this.. so I added one more function :) 
```
// RefineConfig refines the values for clear setting
func RefineConfig(config Config) Config {
	if config.Timeout != DEFAULT_DEPLOYMENT_TIMEOUT {
		config.Timeout = config.Timeout * time.Minute
	}

	if config.PollingInterval != DEFAULT_POLLING_INTERVAL {
		config.PollingInterval = config.PollingInterval * time.Second
	}

	config.StartTimestamp = time.Now().Unix()

	return config
}
```

I wanted to prevent the user mistake (if user specifies 60 not 60s). But I am not sure that if user specifies `60s` then it would be 60s or 60s * time.Second ?

Reply:
When I ran test, 60s (not 60s * time.Second) is applied. I need confirmation :)

Reply:
If user input '60s', It would be 60 seconds. :) You're right.

And, about RefineConfig function.
I think It's good approach! But If user run goployer with polling-interval runtime option like `goployer --polling-interval=30s ...`, It would be override by RefineConfig function.

How about this?
```go
	if config.PollingInterval < time.Second {
		config.PollingInterval = config.PollingInterval * time.Second
	}
```

Reply:
If user input '60s', It would be 60 seconds. :) You're right.

And, about RefineConfig function.
I think It's good approach! But If user run goployer with polling-interval runtime option like `goployer --polling-interval=30s ...`, It would be override by RefineConfig function.

How about this?
```go
	if config.PollingInterval < time.Second {
		config.PollingInterval = config.PollingInterval * time.Second
	}
```

Reply:
Oh!! I changed the code with yours. Then now we can handle the both cases 30 and 30s

Thank you

Reply:
Thank you for your acceptance. :)

Reply:
Thank you for your acceptance. :)